lib/cli/ensure-supported-command.js

```diff
@@ -23,12 +23,52 @@ const getCommandSuggestion = (command, commandsSchema) => {
 export default ({ command, options, commandSchema, commandsSchema }) => {
   // If the command is not found, suggest the closest match
   if (!commandSchema) {
-    const err = new ServerlessError(
+    let errorMessage =
       `Serverless command "${command}" not found.` +
-        `${getCommandSuggestion(command, commandsSchema)} ` +
-        'Run "serverless help" for a list of all available commands.',
-      'UNRECOGNIZED_CLI_COMMAND'
-    );
+      `${getCommandSuggestion(command, commandsSchema)} ` +
+      'Run "serverless help" for a list of all available commands.';
+
+    if (command === 'slstats') {
+      errorMessage = `The "slstats" command was deprecated in v4.`;
+    }
+
+    if (command === 'install') {
+      errorMessage = 'The "install" command was deprecated in v4. Please clone the repo instead.';
```

Comment:
Just to be sure, the message says "deprecated" but are these commands still available or entirely removed? (deprecated suggests they are still here, but we should update our code to move away from them)